app/assets/stylesheets/events.scss

```diff
@@ -308,6 +318,24 @@ label {
       list-style: square;
       font-size: 8pt;
       text-align: left;
+      color: $main-color-5;
+    }
+
+    td.buttons {
```

Comment:
Avoid qualifying class selectors with an element.<br>Selector should have depth of applicability no greater than 2, but was 3